src/backend/distributed/Makefile

```diff
@@ -7,7 +7,7 @@ MODULE_big = citus
 EXTENSION = citus
 EXTVERSIONS = 5.0 5.0-1 5.0-2  \
 	5.1-1 5.1-2 5.1-3 5.1-4 5.1-5 5.1-6 5.1-7 5.1-8 \
-	5.2-1 5.2-2
+	5.2-1 5.2-2 5.2-3
```

Comment:
Also I think we already updated master to `5.2-3`. Could you double check please?